smartsim/_core/launcher/dragon/dragonLauncher.py

```diff
@@ -402,3 +427,15 @@ def _inner(obj: object) -> _SchemaT:
         return obj
 
     return _inner
+
+
+def dragon_cleanup(server_socket: zmq.Socket[t.Any], server_process_pid: int) -> None:
+    try:
+        with DRG_LOCK:
+            DragonLauncher.send_req_as_json(server_socket, DragonShutdownRequest())
+    except zmq.error.ZMQError as e:
+        logger.error(
+            "Could not send shutdown request to dragon server," f" ZMQ error: {e}"
```

Comment:
Looks like this can now be one-lined within the 88 char limit
```suggestion
            f"Could not send shutdown request to dragon server, ZMQ error: {e}"
```